createdisk.sh

```diff
@@ -210,6 +210,12 @@ if [ -z $random_string ]; then
 fi
 VM_PREFIX=${CRC_VM_NAME}-${random_string}
 
+# First check if cert rotation happened successfully and valid for 30 days
```

Comment:
The comment is misleading as this only checks that the certificate will be valid for more than 24 hours starting from the time of the check. You could check for 25*86400 or something like this, and be more explicit in the comment or the log.
`# First check if cert rotation happened. Initial certificate is only valid for 24 hours, after rotation, it's valid for 30 days. We check if it's valid for more than 25 days rather than 30 days to give us some leeway regarding when we run the check with respect to rotation time`